### PR TITLE
Decentralized Effervescent-Token Model and Capital Pricing Tool: Scoping study

### PR DESCRIPTION
**Pull request by taqtiqa-mark**
#### Project Abstract

Prior related W3F Grant: [Token economics survey (2022)](https://github.com/w3f/Grants-Program/blob/master/applications/tokenomics-survey-2022.md)

A scoping study for: A Blockchain capital pricing workbook template and documentation.

Together the workbook and user guide are intended to demonstrate how to:

1. design a (hypothetical) blockchain token with known (theoretical) equilibrium relations,
2. build a workbook model of the equilibrium token dynamics, and
3. construct a capital pricing model for the token.

The target audience is blockchain developers, analysts and users. The tool implementation is blockchain agnostic.

##### Description:
Design a hypothetical token with a known pricing relation to observable variables that influence the token value dynamics.
The token use case and design will strive to be consistent with the constraints and requirements described by [Cong, Lin William, Ye Li, and Neng Wang. “Tokenomics: Dynamic Adoption and Valuation.” The Review of Financial Studies 34, no. 3 (August 2020): 1105–1155.](https://doi.org/10.1093/rfs/hhaa089).
Implement a capital pricing workbook for the so-designed token/coin. The capital pricing framework will be the Black Zero-Beta capital asset pricing model (CAPM).

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 2**:  Up to $30,000, 3 approvals

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ BTC, Ethereum (USDC/DAI), or Polkadot/Kusama (USDT) address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Also, I want to provide some quick feedback here. Sorry, I forgot to reply here before. I think we would be much more interested in supporting https://github.com/w3f/Grants-Program/pull/1854 instead. The main reasons for this are: in general, we are always careful when it comes to funding token-related work, and this application isn't necessarily polkadot specific, whereas the staking application is very focused on our ecosystem. I'm not saying we wouldn't fund it, but I think https://github.com/w3f/Grants-Program/pull/1854 has a higher chance of being accepted, and I'm personally also much more interested in your other application.

**Comment by taqtiqa-mark** [comment]
> The main reasons for this are: in general, we are always careful when it comes to funding token-related work, and this application isn't necessarily polkadot specific, whereas the staking application is very focused on our ecosystem.

Thanks for the feedback. Apologies for the ambiguity - this does assume familiarity with the CLW paper or at least its annotated entry in [Token economics survey (2022)](https://github.com/w3f/Grants-Program/blob/master/applications/tokenomics-survey-2022.md).

I understand the reluctance to be seen funding a particular token. 

 Just to be clear this is not implementing a token that will be launched. It is a scoping study to establish the work that would be required to implement a user/developer tool that will expose the risks of a particular token - that token could already exist in the Dotsama ecosystem, or it could be in development.
The template targets a fairly common use case among parachains and on-demand parachains.

 I'll add the following clarifications if they cause you to revise your current assessment. Let me know if you do change your mind:

- The framework is appropriate for Substrate parachain tokens that:
  - have a fixed supply
  - are not used to fund collating or staking, e.g. on-demand chains, solo chains, permissioned and permissionless chains
  - are expected to have network effects from user adoption
  - are focused on a particular use case rather than being a fiat replacement.

Two use cases I hesitated to put forward are, first, as a parachain pre-launch development tool:
  - can you identify the productivity benefit that is driving your token value?
  - can you measure those things? Directly? Indirectly?
  - do you and your users understand the 'best-estimate' timeframes required for your expected network effects to materialize?
  -  do you and your users understand the range of token values suggested by your data? These tokens are not the-skys-the-limit. They have a limited value range.
 
Second, a tool for users to evaluate an existing token. If it helps I could add an item regarding selecting an existing token in the ecosystem to illustrate this use case - but won't this be controversial? It will appear to be targeting a team.

> I'm personally also much more interested in your other application.

That is on me. This tool provides similar functionality but at a broader/higher level. Rather than just asking: Is staking working as they say? This tool will be asking (within the constraints of a model): Is this token working as expected?

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the clarification here, and sorry for the late reply. I would still be much more interested in your other application, but let me share this with the rest of the team as well. So it's not just based on my opinion.

**Comment by taqtiqa-mark** [comment]
> I would still be much more interested in your other application

Is this an either-or choice? 

If so the other application is more important - right now there is zero help for developers creating a token economy - yet all the tools are provided to do so.  
This is a recipe for disasters ... like the Acala stablecoin.

**Comment by Noc2** [comment]
We usually support only one application at a time per person/team unless it's a bigger team.

**Comment by taqtiqa-mark** [comment]
Understood. Please evaluate this application as the higher priority - IMO. If it gets declined we can move on.

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/referendum-234-data-availability-retroactive-funding-and-on-chain-invoices/3421/37

**Comment by dsm-w3f** [comment]
@taqtiqa-mark thank you for your application. Is this application a research or development one? I notice that the application is not compliant with any of the two templates that we have ([software development](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) or [research](https://github.com/w3f/Grants-Program/blob/master/applications/application-template-research.md)). Please select one and make the adjustments to comply with the template. Give special attention to the milestone deliveries table. Let me know when I can take another look at this application.

**Comment by taqtiqa-mark** [comment]
> @taqtiqa-mark thank you for your application. Is this application a research or development one? 

Development.

> I notice that the application is not compliant with any of the two templates

Correct - per the discussion above @Noc2 and 'the team' are assessing if this type or tool is in the scope of the grant program.  If it is I'll provide the additional detail taking into account any feedback.

**Review by Noc2 — CHANGES_REQUESTED** [review]
I just saw that you requested my review here. I took another look at your previous grant and the delivery, and to be honest, I don't see a clear benefit regarding this project for the Polkadot ecosystem or somehow picking up this work and actually implementing a unique token design based on it. But I also shared the application with @jonasW3F, who might be in a better position to judge it. And I also pinged the rest of the team again.

**Comment by taqtiqa-mark** [comment]
> I just saw that you requested my review here. I took another look at your previous grant and the delivery, and to be honest, I don't see a clear benefit regarding this project for the Polkadot ecosystem ....

@Noc2 , I Just wanted to clarify a couple of things. By persisting I'm not discounting your objections. And I have taken time reviewing [other grants](https://github.com/w3f/Grants-Program/pull/1813), in areas I am expert, to try to understand [what you consider the community needs](https://github.com/w3f/Grants-Program/pull/1813#pullrequestreview-1511030874):

> > Especially since the grants program primarily supports the development of a PoC or something similar and not production-ready software.

> or somehow picking up this work and actually implementing a unique token design based on it. 

Yes that is one use case, which I describe as pre-launch.  I do point out the validation use case, which I describe as post-launch. I believe there is [some interest](https://forum.polkadot.network/t/how-overvalued-is-polkadot-dot) in the post-launch use case. But PoC for this use case maybe out of scope?

> But I also shared the application with @jonasW3F, who might be in a better position to judge it. And I also pinged the rest of the team again.

Thanks, I'll make a couple of more revisions. Then if the application is still does not persuade the team we can safely say this line of tooling is categorically out of scope.  That is important because the published proof of token dynamics is one of the simplest and least risky - things get exponentially more difficult, and more risky, from there.

**Comment by dsm-w3f** [comment]
@taqtiqa-mark As I commented [here](https://github.com/w3f/Grants-Program/pull/1848#issuecomment-1658116780), the deliverables table is still not compliant with our template. If you want to go forward with this application, please adjust it, so we can mark it as ready to review by the committee.

**Comment by taqtiqa-mark** [comment]
Hi I'll respond inline: 
>  Hi @taqtiqa-mark

> thanks for your interest in working on this for the ecosytem. I share @Noc2 concern that this might not be suitable for a Web3 Grant as one of your main goals is to come up with a pricing model of Parachain token.

For calrity the pricing model has already been developed in the published paper cited.  I certainly am not "coming up with a pricing model of Parachain token".

> The other goals will certainly also be very much related to overall market dynamics.

That statement on market dynamics isn't clear to me.  But I think I understand the reservations: the W3F doesn't want to be seen to be promoting a particular token model.

> We generally abstain from giving any guidelines about the financial aspects around DOT or parachains. 

Understood, I very much share that reservation (see below).  Hence this proposal was generic.

> I do, however, think that it might be a valuable contribution to the ecosystem, so my guess would be that the Treasury is a better place to ask for funding. 

I'll certainly consider that.

> On a personal note: as you certainly are confident that your contribution will be valuable, there are many new projects out there that are very interested in getting help with their token design. 

I bear that in mind.

> I do, however, appreciate your broader perspective of coming up with open guidelines for the ecosystem. Again, this sounds more suited for the Treasury.

Yes, an open tutorial is certainly the primary motivation.

>  As a sidenote. I read your delivery of Milestone 3 from the last grant. I already expressed my concern of relevance at the time. After having reviewed your manuscript, I feel confirmed in my initial estimate: 

I think you're being selective here.  You inital estimates was:

> > ... rigorous (academic) token economic research is mostly absent. 

We now know that is a profound misconception. One I shared in.  But I was more optimisitc than you and that optimism has been bourne out more than I'd expected - the Sockin papers are particularly encouraging - and possibly, in my mind, explains the SEC more aggressive stance.  Their staff can't be excused the level of irnorance I had. 

And now we can point to this paper in the ecosystem showing this research is very rigorous and has been well known for several years in the form of discussion papers, and published in the most prominant journals for going on three years.

> the only reference to anything related Polkadot is Table 4 that simply categorizes the parachain tokens into three rather obvious types.

Again your prior expectation was:
 
> > I'd not expect much other tokens than utility tokens that pay for transaction fees, are used in governance for voting and/or might be used for staking. Other token types are virtually absent as they'd constitute clear securities.

I'll need to think about how to phrase things in the paper so that it is clearer what those authors are saying - that your understanding (stated above) is very near back to front.  
The difficulty here is the economic definition of a security vs the legal definition.  The courts haven't paid to much attention to the economic definitions - preferring to 'look behind the curtain'.

All of the analysis done applies to the Polkadot ecosystem. Unfortunately, there was only the Equillibirum token/chain that contained any material that could be identified as being in the framework reviewed here.  It is valuable to know that fact.

> Other than that, the document just lists a few economic concepts, in my opinion misses making the reader aware of the relevance between each them,

That is a fair point.  The relevance between them is set out in the 1,200+ pages of Lunqvist and Sargeant.  I'll make this explicit.

> and summarizes 6 (relevant) papers. Here, again, the reader is not educated about the relevance of the studies to the specific application in Polkadot. 

Yes that is correct.  To do so would be "to come up with a pricing model of Parachain token"  and would be to provide "guidelines about the financial aspects around DOT or parachains"

> I doubt that giving this document to any parachain team has them learn something about their token economics. 

This is a problem.  

For example, I introduce the term 'breakable token' that is understandable to a lay reader. That is it is not the formal economic charaterization.  I do make clear which token 'features' or 'functions' make a token breakable.  

It does appear you are trying to get me to say, in this scoping review, that token X is best modelled, or most closely resembels a 'breakable' token, or token Y has characterisitics that are consistent with an 'effervescent' token.  

Can you elaborate how what you are asking is different from asking me "to come up with a pricing model of Parachain token"  and to provide "guidelines about the financial aspects around DOT or parachains".  

It may help to use a hypothetical chain/token.  And to describe, or mock-up, some concrete artifact you had expected to see that is not present.
   
> You might argue that applying the theories from the papers to the parachain tokens is out of scope of the delivery. 

It very clearly is out of scope.  Ironically, for the very reasons you object to this proposal. To do so would be "to come up with a pricing model of Parachain token" for some particular team  and to provide "guidelines about the financial aspects around DOT or parachains".

I did address explicitly in the paper why that is not done - I do not know what a developer (team) has in their mind's eye for their token.  The subtle details matter greatly.

>  But, I would still have loved to see some tangible implications for practitioners in the last (now fully delivered) grant. 

As I said I'm happy to try to do that.  

Can you indicate what you have in mind that is not a backdoor way "to come up with a pricing model of Parachain token"  and to provide "guidelines about the financial aspects around DOT or parachains"?
 
> Also in this grant proposal, I'd love to see some upfront work giving a glimpse on the usefulness of the final study.

Regarding the details of the previous grant mentioned above please open issues against the previous grant addressing the detail and artifacts you believe I should have delivered.  I'll do my best to address those shortcomings - within the constraints that we seem to agree on: they should not be a backdoor way "to come up with a pricing model of Parachain token"  and are not to provide "guidelines about the financial aspects around DOT or parachains".

I don't propose to revise this application further.  I'll await reject/accept decision then we can move on.

**Comment by taqtiqa-mark** [comment]
> @taqtiqa-mark As I commented [here](https://github.com/w3f/Grants-Program/pull/1848#issuecomment-1658116780), the deliverables table is still not compliant with our template. 

Yes, as indicated previously by @Noc and now @jonasW3F there is concern this topic is out of scope of the program. Unless that changes it appears we are headed to rejecting the PR as out of scope.  Then we can move on.

> If you want to go forward with this application, please adjust it, so we can mark it as ready to review by the committee.

IIUC, we have to clear the first hurdle of establishing the topic is within scope.

File: applications/decentralized-capital-pricing-tool-scope.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
+# Decentralized Effervescent-Token Model and Capital Pricing Tool: Scoping study
+
+- **Team Name:** Mark Van de Vyver [PhD](https://www.student.uwa.edu.au/course/award-verification-service?family=van+de+vyver&family_partial=on&given=mark&search=Search)
+- **Payment Address:** 15WoVugRLfBAogS1FBmhzxVv8WZqZK1z4KKHALijTZfdVcCm (USDT)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
```
Comment (semuelle) [inline applications/decentralized-capital-pricing-tool-scope.md:5]:
```suggestion
- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
```

30k is Level 2.

**Comment by taqtiqa-mark** [comment]
In light of the experience with the grant [Token economics survey (2022)](https://github.com/w3f/Grants-Program/blob/master/applications/tokenomics-survey-2022.md). Specifically, the attempt, on the brink of accepting Milestone 3, to bring within scope material that was clearly defined as out of scope, and the attempt to provide guidance to block chain teams. Details here:

https://github.com/taqtiqa-mark/tokenomics-survey-2022/issues/4

And given the complexity and subtlety of the analysis that is the subject of this application, prior experience dictates it is prudent to withdraw this application.

**Comment by semuelle** [comment]
Hi @taqtiqa-mark. I respect your decision to withdraw your applications. But could you elaborate what you mean by 

> bring within scope material that was clearly defined as out of scope

? Are you referring to the ongoing grant, or the application?

**Comment by taqtiqa-mark** [comment]
> Are you referring to the ongoing grant, or the application?

The current grant, which was all but signed off until this application was discussed.

**Pull request closed without merging**